x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/optimizer/Optimizer.java

```diff
@@ -108,51 +102,6 @@ public LogicalPlan optimize(LogicalPlan verified) {
         return Arrays.asList(substitutions, operators, constraints, operators, ordering, local, label);
     }
 
-    private static class ReplaceWildcards extends OptimizerRule<Filter> {
```

Comment:
Can this be kept around but not actually used?

Reply:
It can be but considering it worked when dealing with operators and `*` within a string, unless this scenario applies again this rule will not be needed.